### PR TITLE
Convert Search Page. Do listings pagination.

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

File: src/app/pages/SearchPage/styles.less [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  &__searchBar {
+    height: 6 * @grid-size;
+
+    .themeify({
+      border-bottom: @theme-border;
+    });
+  }
+
+  &__loading {
+    position: absolute;
```
Comment (ajacksified) [inline src/app/pages/SearchPage/styles.less]: this forces the loading icon to the far left side; nuke this rule?

**Comment by ajacksified** [comment]
The `after` query param doesn't get added to the `Next >` button or the API request.

💅

**Comment by schwers** [comment]
`after` is appearing for me -- I'll try resetting node modules and doing a clean install. Can you send me the urls your app is making for `Next >` buttons?

Locally I'm seeing these urls for search: `/search?q=react&after=t3_4fsquv&page=1&type=link`, communities search:`/search?q=react&type=sr&after=t5_3c0ga&page=1`, and listings: `/r/all?count=25&page=1&after=t3_4l4e85`

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**